apps/src/codebridge/FileBrowser/index.tsx

```diff
@@ -649,13 +649,13 @@ export const FileBrowser = React.memo(() => {
 
         validateInput: (destinationFolderName: string) => {
           try {
-            const folderId = findFolder(destinationFolderName.split('/'), {
+            const parentId = findFolder(destinationFolderName.split('/'), {
```

Comment:
`folderId` was an ambiguous name, since it's actually checking against the potential _parent_ folder.